code/_onclick/hud/ai.dm

```diff
@@ -0,0 +1,148 @@
+/mob/living/silicon/ai
+	hud_type = /datum/hud/ai
+
+/datum/hud/ai/FinalizeInstantiation()
+
+	if(!isAI(mymob))
+		return
+
+	var/mob/living/silicon/A = mymob
+
+	adding = list()
+	adding += new /obj/screen/ai_button(
```

Comment:
Personally I really dislike having the args for new() spread out over so many lines. I can sort of see why it's being done, due to the proc lists, but it feels very weird. Not going to block the PR over it but.. yeah.